tests/integration/standard/test_client_warnings.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2013-2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from cassandra.query import BatchStatement
+from cassandra.cluster import Cluster
+
+from tests.integration import use_singledc, PROTOCOL_VERSION
+
+
+def setup_module():
+    use_singledc()
+
+
+class ClientWarningTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        if PROTOCOL_VERSION < 4:
+            raise unittest.SkipTest(
+                "Native protocol 4,0+ is required for client warnings, currently using %r"
+                % (PROTOCOL_VERSION,))
+        cls.cluster = Cluster(protocol_version=PROTOCOL_VERSION)
+        cls.session = cls.cluster.connect()
+
+        cls.session.execute("CREATE TABLE IF NOT EXISTS test1rf.client_warning (k int, v0 int, v1 int, PRIMARY KEY (k, v0))")
+        cls.prepared = cls.session.prepare("INSERT INTO test1rf.client_warning (k, v0, v1) VALUES (?, ?, ?)")
+
+        cls.warn_batch = BatchStatement()
+        # 213 = 5 * 1024 / (4+4 + 4+4 + 4+4)
+        #        thresh_kb/ (min param size)
+        for x in range(213):
+            cls.warn_batch.add(cls.prepared, (x, x, 1))
+
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.cluster.shutdown()
+
+    def test_warning_basic(self):
+        """
+        Test to validate that client warnings can be surfaced
+
+        @since 2.6
+        @jira_ticket PYTHON-315
+        @expected_result valid warnings returned
+        @test_assumptions
+            - batch_size_warn_threshold_in_kb: 5
+        @test_category queries:client_warning
+        """
+        future = self.session.execute_async(self.warn_batch)
```

Comment:
There should be a newline after the end of the comment block, and before the beginning of code.